CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file.
 
+## 2.1.0
+
+- fix: disabled code editor and client cache to prepare for release in [#448](https://github.com/grafana/iot-sitewise-datasource/pull/448)
```

Comment:
anything related to code editor can be omitted since it was disabled before released

Reply:
how about we include only the ones marked `feat`?